Source/Scene/GroundPrimitive.js

```diff
@@ -240,7 +240,7 @@ define([
 
         var readOnlyAttributes;
         if (defined(this.geometryInstances) && isArray(this.geometryInstances) && this.geometryInstances.length > 1) {
-            readOnlyAttributes = readOnlyInstanceAttributesScratch;
```

Comment:
Are you sure you want to remove the scratch?  These are used for performance (and maybe memory usage here).  I'm not sure how important this case is.  Ask @bagnell.